commands/util.go

```diff
@@ -15,12 +22,29 @@ func loadSafe() (*safe.Safe, error) {
 		return nil, err
 	}
 
-	backendClient, err := backends.New(config.Storage)
+	home, err := homedir.Dir()
+	if err != nil {
+		return nil, err
+	}
+
+	config := safe.NewConfig()
+	config.Encryption = crypto.NewDefaultConfig()
+
+	configFile := fmt.Sprintf(defaultConfigFileTmpl, home)
+	if _, err := os.Stat(configFile); os.IsNotExist(err) {
+		// TODO(): No config found, should we create one?
+	} else {
+		if _, err := toml.DecodeFile(configFile, &config); err != nil {
+			return nil, err
+		}
+	}
+
+	backendClient, err := backends.New(&config.Storage)
 	if err != nil {
 		return nil, err
 	}
 
-	cryptoClient, err := crypto.New(config.Encryption)
+	cryptoClient, err := crypto.New(&config.Encryption)
```

Comment:
You also need to update `safe/utils_safe.go` to pass the pointer to `crypto.New`. Tests are currently broken.